storage/grpc_metrics_test.go

```diff
@@ -34,15 +34,19 @@ func TestMetricFormatter(t *testing.T) {
 	}
 }
 
-func TestNewPreparedResource(t *testing.T) {
+func TestStorageMonitoredResource(t *testing.T) {
 	ctx := context.Background()
 	for _, test := range []struct {
 		desc               string
+		project            string
+		api                string
 		detectedAttributes []attribute.KeyValue
 		wantAttributes     attribute.Set
 	}{
 		{
-			desc: "default values set when GCP attributes are not detected",
+			desc:    "default values set when GCP attributes are not detected",
+			project: "project-id",
```

Comment:
project and api seem the same in all test cases, should they just be removed as fields here? Or did you want to test other values?